From 9d8045f0f2e4282c6cc6343235e9735c8cfcee91 Mon Sep 17 00:00:00 2001
From: David Bauer <blocktrron@users.noreply.github.com>
Date: Sat, 29 Dec 2018 23:32:02 +0100
Subject: [PATCH] ramips-mt76x8: remove broken flag (#1608)

This commit removes the broken flag from all devices in the mt76x8
subtarget.

The stability of the mt76 driver for the mt7628 and mt7612 has greatly
improved in the last half-year. It might be still behind ath9k and
ath10k but it is suitable for daily use.

This affects the following devices:

 - GL.iNet MT300N v2
 - TP-Link Archer C50 v3
 - TP-Link TL-WR841 v13
---
 docs/index.rst        | 9 +++++++++
 targets/ramips-mt76x8 | 5 +----
 2 files changed, 10 insertions(+), 4 deletions(-)

diff --git a/docs/index.rst b/docs/index.rst
index 69b2cd1f..e06b5a3d 100644
--- a/docs/index.rst
+++ b/docs/index.rst
@@ -376,6 +376,15 @@ ramips-mt7621
 ramips-mt76x8
 ^^^^^^^^^^^^^
 
+* GL.iNet
+
+  - GL-MT300N v2 [#80211s]_
+
+* TP-Link
+
+  - TL-WR841N v13 [#80211s]_
+  - Archer C50 v3 [#80211s]_
+
 * VoCore
 
   - VoCore2 [#80211s]_
diff --git a/targets/ramips-mt76x8 b/targets/ramips-mt76x8
index 45e00977..bc1fdf95 100644
--- a/targets/ramips-mt76x8
+++ b/targets/ramips-mt76x8
@@ -1,10 +1,9 @@
-if [ "$BROKEN" ]; then
-
 # GL.iNet
 
 device gl-mt300n-v2 gl-mt300n-v2
 factory
 
+
 # TP-Link
 
 device tp-link-archer-c50-v3 tplink_c50-v3
@@ -15,8 +14,6 @@ device tp-link-tl-wr-841n-v13 tl-wr841n-v13
 factory
 extra_image -squashfs-tftp-recovery -bootloader .bin
 
-fi
-
 
 # VoCore 2
 
-- 
2.17.1

